migrate/v0_15/migrate.go

```diff
@@ -174,14 +175,50 @@ func Committee(genesisState v0_14committee.GenesisState) v0_15committee.GenesisS
 			baseStabilityCom := stabilityCom.SetPermissions(newStabilityCommitteePermissions)
 			newStabilityCom := v0_15committee.MemberCommittee{BaseCommittee: baseStabilityCom}
 			committees = append(committees, newStabilityCom)
-		} else {
+		case 2:
 			safetyCom := comtypes.NewMemberCommittee(com.ID, com.Description, com.Members,
 				[]v0_15committee.Permission{v0_15committee.SoftwareUpgradePermission{}},
 				com.VoteThreshold, com.ProposalDuration, v0_15committee.FirstPastThePost)
 			committees = append(committees, safetyCom)
 		}
 	}
 
+	// Initialize hard governance committee
+	hardGovCom := comtypes.TokenCommittee{}
+	hardGovCom.ID = 3
+	hardGovCom.Description = "Hard Governance Committee"
+	hardGovCom.Members = []sdk.AccAddress{sdk.AccAddress("kava1e0agyg6eug9r62fly9sls77ycjgw8ax6xk73es")}
+	hardGovCom.VoteThreshold = sdk.MustNewDecFromStr("0.5")
+	hardGovCom.ProposalDuration = time.Duration(604800000000000)
+	hardGovCom.TallyOption = v0_15committee.FirstPastThePost
+	hardGovCom.Quorum = sdk.MustNewDecFromStr("0.33")
+	hardGovCom.TallyDenom = "hard"
```

Comment:
`Quorum` and `TallyDenom` are not getting preserved when marshaling to JSON.

Reply:
The issue was that the `Committee` interface's `SetPermissions` function always returned `BaseCommittee`, which caused the extra fields to be dropped from TokenCommittee in the migration after permissions were set. Fixed in https://github.com/Kava-Labs/kava/pull/941/commits/7ef2087dde9d915a1b79ddcbf6d95c1d4802e041 by updating the return type to the `Committee` interface and adding `SetPermissions` methods to each sub-committee type.